engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/compile/ProcessValidatorSpec.scala

```diff
@@ -100,6 +100,7 @@ class ProcessValidatorSpec extends FunSuite with Matchers with Inside {
     ClassExtractionSettings.Default
   )
 
+  //todo: implement a test after modifying valid static methods list so there is a valid static method returning a Boolean
```

Comment:
Why you've removed a test case? I don't understand this comment. Are we really blocked by sth? Can't we modify a bit test case/test setup to make writting this test possible ?